### ISSUE TITLE
Cannot insert explicit value for identity column in table 'Movies' when IDENTITY_INSERT is set to OFF.

### ISSUE THREAD

**Issue by Razvan30** [mosh-hamedani/vidly-mvc-5#49]
Please help with this error!. I try to save new movie an i got this error message: "Cannot insert explicit value for identity column in table 'Movies' when IDENTITY_INSERT is set to OFF.  
![capturemovieerror](https://user-images.githubusercontent.com/20680575/44958620-7f3c3580-aeeb-11e8-98d5-7ba2b63fcdb8.PNG)
"

Thanks!

**Comment by ahmednadeem** [comment]
In up() method of migration PopulateMembershipTypes execute the follwing command while populating the table
Sql("SET IDENTITY_INSERT MembershipTypes ON")

**Comment by ahmednadeem** [comment]
In up() method of migration execute the follwing command while populating the table
Sql("SET IDENTITY_INSERT Movies ON")

**Comment by Razvan30** [comment]
OK, done... u save my life!

**Comment by ahmednadeem** [comment]
No that was compilation error, i had to create new project

**Comment by Razvan30** [comment]
Yes RyanFlemming, your answer solve my issue. I tried to add :   Sql("SET IDENTITY_INSERT Customer ON");
            Sql("INSERT INTO Customer (Id, Name, IsSubscribedToNewsletter, MembershipTypeId, Birthdate) VALUES (2 , 'Razvan30', 'True', 1,  2/2/2011)");

**Comment by Razvan30** [comment]
Thanks! Now I understand why  i get this error.

**Comment by iHamzaKhanzada** [comment]
@Razvan30 , if you have solved the error, please mark this issue as closed
